Update ide0073.md (#23747)
dotnet/docs: docs/fundamentals/code-analysis/style-rules/ide0073.md

|Property|Value|
|-|-|
| **Option name** | file_header_template

> | **Option values** | non-empty string - Prefer the string as required file header, replacing `{fileName}` placeholders (if any) with the actual file.<br /><br /> `unset` or empty string - Do not require file header. |

Reviewer: ```suggestion
| **Option values** | non-empty string, optionally containing a `{fileName}` placeholder - Prefer the string as required file header.<br /><br /> `unset` or empty string - Do not require file header. |
```

| **Option values** | non-empty string - Prefer the string as required file header.<br /><br /> `unset` or empty string - Do not require file header. |
| **Default option value** | `unset` |

> You can include `{fileName}` as a placeholder for the file name.

Reviewer: Should we put this alongside the non-empty string description for the option value?
